torchci/pages/metrics.tsx

```diff
@@ -1133,6 +1133,18 @@ export default function Page() {
             groupByFieldName={"job_name"}
           />
         </Grid>
+        <Grid item xs={12} height={ROW_HEIGHT}>
+          <TimeSeriesPanel
+            title={"Percentage of jobs rolled over to Linux Foundation"}
+            queryName={"lf_rollover_percentage"}
+            queryCollection={"metrics"}
+            queryParams={[]}
+            granularity={"day"}
+            timeFieldName={"bucket"}
+            yAxisFieldName={"percentage_lf"}
+            yAxisRenderer={(value) => value + "%"}
```

Comment:
the rendering didn't work.  You might need to convert `value` to a string first

<img width="179" alt="image" src="https://github.com/user-attachments/assets/59550284-b291-46d8-b711-305156502dca">

Reply:
This is interesting, as it works on the side, but will fix

Reply:
Really? The vercel preview didn't have it working on the side either. Sounds like worked locally for you though

Reply:
Vercel still doesn't show render the value :/ 
See https://torchci-git-fork-palic-palic-lfpercent-fbopensource.vercel.app/metrics